app/views/admin/data_imports/show.html.erb

```diff
@@ -1,27 +1,42 @@
-<div class="flex flex-col w-full mx-auto">
-  <h1 class="w-full max-w-prose mx-auto text-center mb-3">Data Import</h1>
-  <div class="w-full max-w-prose mx-auto mb-4">
-    <div class="mb-4">
-      <div class="show-label">Description</div>
-      <div class="show-field"><%= @data_import.description %></div>
-    </div>
-    <div class="mb-4">
-      <div class="show-label">File</div>
-      <div class="w-full mb-1">
-        <%= link_to @data_import.file.filename.to_s, url_for(@data_import.file), class: "text-blue-600 hover:text-blue-800 visited:text-purple-600" %>
-      </div>
-    </div>
-    <% if @data_import.occupation_standard %>
-      <div class="mb-4">
-        <div class="show-label">Occupation standard</div>
-        <div class="w-full mb-1">
-          <%= link_to @data_import.occupation_standard_title, admin_occupation_standard_path(@data_import.occupation_standard), class: "text-blue-600 hover:text-blue-800 visited:text-purple-600" %>
-        </div>
-      </div>
-    <% end %>
-  </div>
-  <div class=" max-w-prose mx-auto flex text-center space-x-2">
-    <%= link_to "Edit data import", edit_admin_source_file_data_import_path(@source_file, @data_import), class: "btn-primary" %>
-    <%= button_to "Delete data import", admin_source_file_data_import_path(@source_file, @data_import), method: :delete, data: {turbo_method: :delete}, class: "btn-primary" %>
+<%# erblint:counter GitHub::Accessibility::LandmarkHasLabelCounter 1 %>
```

Comment:
@FerPerales is this an erblint exception? Does Administrate just not handle this case well?